E-BANKING WORKSPACE.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file is specific for your setup, add it to gitignore, as it's not relevant for the project